app/src/androidTest/java/com/android/unio/components/authentication/PictureSelectionToolTest.kt

```diff
@@ -0,0 +1,86 @@
+package com.android.unio.components.authentication
+
+import android.net.Uri
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.test.filters.LargeTest
+import com.android.unio.TearDown
+import com.android.unio.model.strings.test_tags.PictureSelectionToolTestTags
+import com.android.unio.ui.components.PictureSelectionTool
+import com.google.firebase.Firebase
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.auth
+import com.google.firebase.auth.internal.zzac
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
```

Comment:
Watch out mixing mockk and mockito and try to stick to only mockk

Reply:
sure !